### PR TITLE
Sidebar icons

### LINKED ISSUE #127: Add sidebar icons and HREFs
- add kashti logo
- clicking kashti logo takes you to main page
- help icons exist: 1) docs and 2) support
- hover over docs and support shows text (probably use CSS for effect)

### PR DESCRIPTION
**Pull request by seanknox**
- add kashti logo
- clicking kashti logo takes you to main page
- help icons exist: 1) docs and 2) support
- ~hover over docs and support shows text (probably use CSS for effect)~ (i didn't see this in use at the https://azure.github.io/kashti/#!/ demo)

Closes #127

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1,97 +1,15 @@
...
+@import 'ionicons';
 
-// kashti sidebar styles
+@import 'global_variables';
 
-#sidebar-wrap {
-  flex: 0 0 56px;
-  margin: 0;
+.full-height {
+  flex: 0 0 3.5em;
```
Comment (seanknox) [inline src/styles.scss]: @flynnduism I changed this from `#sidebar-wrap` to a class in an attempt to be semantic. I know this class ensures the element will use the full height of the browser. I wonder if we could extract the flex property to it's own class that's meaningful semantically. 

thus the resulting HTML would be something like

```html
<div class="column full-height flex-right-margin-10" data-off-canvas data-position="left">
    <div id="sidebar">
```
(`flex-right-margin-10` doesn't really make sense...not sure how to describe the flex property here semantically)

File: src/app/dashboard/dashboard.component.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
+@import "../../global_variables";
+
+// kashti sidebar styles
+#sidebar {
+  background: #37b0e7 url('../../assets/images/topo-pattern-white.png') repeat 20px 0;
+  background-size: 376px auto;
+  width: 56px;
```
Comment (seanknox) [inline src/app/dashboard/dashboard.component.scss]: Would like to convert px statements like this to `em` or %; in this case we'd also want to center the logo and icons. Thoughts?

Comment (flynnduism) [reply src/app/dashboard/dashboard.component.scss]: Fine with me. It's based on the logo width, a one-off measurement in the UI.

File: src/styles.scss (hunk 1 above)
Comment (flynnduism) [reply src/styles.scss]: 👍 

Changing the id to a class is better.
Splitting the .full-height out to be either a class of it's own or a mixin makes sense. 

Is `flex-right-margin-10` to make re-usable gutter elements?

Comment (seanknox) [reply src/styles.scss]: > Is flex-right-margin-10 to make re-usable gutter elements?

Yeah...I was thinking about how to convey (in English) what this does:

```css
flex: 0 0 3.5em;
```

In this case, does the above specify the initial main size of a flex item?

Comment (flynnduism) [reply src/styles.scss]: This the flexbox equivalent to width for the column. Modularize it any way you want, it's a singular exception to the existing grid so not a common element.

**Comment by flynnduism** [comment]
This gets a solid LGTM

**Pull request merged**